src/config/lang/english.go

```diff
@@ -337,6 +337,8 @@ const (
 	CmdPrepareFlagGitAccount    = "User or organization name for the git account that the repos are created under."
 	CmdPrepareFlagKubeVersion   = "Override the default helm template KubeVersion when performing a package chart template"
 
+	CmdPrepareLintShort = "Verifies the package schema"
```

Comment:
I think we can relatively easily make this do a little more if we refactor packager to have some of its warnings generated from `validator`.  This doesn't _have_ to be in this PR, but if it isn't, a new issue should be created to add it that gets worked on next.

Reply:
I'd rather do it in the next PR just created - https://github.com/defenseunicorns/zarf/issues/2149